dbinit/data/productionActivityCategories.json

```diff
@@ -155,7 +594,7 @@
       "id": "27395d17-7022-4f71-9daf-f431ff4f11e8",
       "localizableName": {
          "en-US": "Social",
-         "nl-NL": "Social media"
+         "nl-NL": "Sociale Netwerken"
```

Comment:
Should this be with two capital letters?

Reply:
I had the same question but didn't raise it to Cornelis. Now I did and he agreed to make it lowercase.